app/code/Magento/Customer/view/adminhtml/templates/tab/view/personal_info.phtml

```diff
@@ -15,52 +15,52 @@ $allowedAddressHtmlTags = ['b', 'br', 'em', 'i', 'li', 'ol', 'p', 'strong', 'sub
 ?>
 <div class="fieldset-wrapper customer-information">
     <div class="fieldset-wrapper-title">
-        <span class="title"><?= $block->escapeHtml(__('Personal Information')) ?></span>
+        <span class="title"><?= $escaper->escapeHtml(__('Personal Information')) ?></span>
```

Comment:
It seems like escaper is not defined in this file

Reply:
fixed